app/react-components/components/SurvivalPlotWrapper.js

```diff
@@ -41,13 +41,29 @@ const styles = {
   },
 };
 
+function toMap(d) {
```

Comment:
Maybe make this a little more general or add an extra argument for sorting?

function toMap (d, sort = sortById) {
  return new Map(Object.entries(d).sort(sort)
}
just a thought